libs/openchallenges/home/src/lib/challenge-host-list/challenge-host-list.component.html

```diff
@@ -1,6 +1,6 @@
 <section id="organizations" class="mat-typography">
   <div class="container">
-    <h3 class="section-title">Discover Platforms and Organizations</h3>
+    <h3 class="section-title">Discover Organizations</h3>
```

Comment:
We can use this opportunity to define what "Organization" means in the context of OC. E.g.

```suggestion
    <h3 class="section-title">Discover the Organizations that contributes to Challenges</h3>
```